resources/interceptor-service-open-api-v1.yaml

```diff
@@ -29,6 +29,7 @@ info:
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
+basePath: /api/v1
```

Comment:
Can this affect other samples? @renuka-fernando @Amila-Rukshan

Reply:
I will build and run other samples to confirm it. Other examples are reading it from servers url I guess

Reply:
In OpenAPI 3.0, the `servers` field defines base URLs. `servers` field replaces the host, basePath, and schemes keywords used in OpenAPI 2.0.
https://swagger.io/docs/specification/api-host-and-base-path/

Reply:
Just checked the spec in the swagger editor, it prompts an error when `basePath: /api/v1` is included.
https://editor.swagger.io/

Reply:
added another API definition file with swagger 2.0 to avoid this issue.

Reply:
added a note to the `README.md` file for this section